src/scenes/SearchResults/FoundMovies/ResultBody.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import ResultItem, { ResultItemProps } from './ResultItem';
+

```

Comment:
You could add this styling into the `search.tsx` file and do the reult loop there, so that this comp could be deleted.

Reply:
I agree, first I have to fix webpack build with css, because it is currently failing with styles.
I am going to fix this as well in the next: task 4 exercise.